code/modules/clothing/under/xenos/seromi.dm

```diff
@@ -34,134 +34,252 @@
 	icon_state = "seromi_rainbow"
 
 /obj/item/clothing/under/seromi/undercoat
+	name = "you shouldnt be seeing this"
 	desc =  "A Teshari traditional garb, with a modern twist! Made of micro and nanofibres to make it light and billowy, perfect for going fast and stylishly!"
```

Comment:
Typically for prototypes that shouldn't be available in game, the name is set to something generic, and the description reflects the fact that it shouldn't be available. So we don't get an ahelp about something along the lines of
> This is X
> They are wearing a you shouldnt be seeing this

or
> What in the hells is a 'you shouldnt be seeing this'?

Reply:
ok i can change that, i mostly changed that for myself when i noticed an odd "under" option in the selection and i recompiled to see that it was the base item